src/pdfpc.vala

```diff
@@ -73,6 +78,9 @@ namespace pdfpc {
             {"list-actions", 'L', 0, 0,
                 ref Options.list_actions,
                 "List actions supported in the config file(s)", null},
+            {"list-monitors", 'M', 0, 0,
+                ref list_monitors,
```

Comment:
For the sake of consistency: All other stuff resides in Options. This should go there also (I think)

Reply:
Actually, I was thinking of moving two more flags out of `Options` - `list_actions` and `version`. These three together are quite different from the rest; they are not options proper, but rather enable a very specific behavior, printing some info and exit. But it is not principal, if you prefer all to be under `Options`, fine.

Reply:
No, I just want a schema which is in Options and what not :) If you want to move out more stuff that's fine.